boot/freeldr/freeldr/ntldr/winldr.c

```diff
@@ -74,6 +74,25 @@ NtLdrOutputLoadMsg(
     }
 }
 
+PACPISDTHeader
+FindAcpiTable()
+{
+    if (!AcpiPresent)
+        return NULL;
+
+    PRSDP_DESCRIPTOR Rsdp = FindAcpiBios();
+    if (!Rsdp)
+        return NULL;
+
+    BOOLEAN is64bitsAcpi = (sizeof(PVOID) > 4) ? (Rsdp->revision > 1) : FALSE;
+    PRSDT_DESCRIPTOR Rsdt = (PVOID)((ULONG_PTR)Rsdp->rsdt_physical_address);
+    PXSDT_DESCRIPTOR Xsdt = (PVOID)((ULONG_PTR)Rsdp->xsdt_physical_address);
+    PVOID OutputPointer = is64bitsAcpi ? (PVOID)Xsdt : (PVOID)Rsdt;
+
+    TRACE("Found acpi table at 0x%p address\n", OutputPointer);
```

Comment:
```suggestion
    TRACE("ACPI table at 0x%p\n", OutputPointer);
```
NB:
Maybe report `is64bitsAcpi` too?

Reply:
No. Report only the `OutputPointer` and i changed the `TRACE` Message.